cmd/privateendpoints-setuplink.go

```diff
@@ -72,7 +81,7 @@ var privateEndpointsSetupLinkCmd = &cobra.Command{
 			zap.String("service-name", pe.ServiceName),
 			zap.String("private-dns", pe.PrivateDNS))
 
-		if instanceId != "" {
+		if strings.Contains(strings.ToLower(cloudCluster.CloudProvider), "aws") {
```

Comment:
nit: Considering we use this cloudProvider detail from multiple places, it might be useful to force it to known values when we read it instead of needing to do that at every comparison point.  Then it can be a simple comparison.

Reply:
:+1: